src/app/_blocks/Subscribe/index.tsx

```diff
@@ -1,11 +1,19 @@
+import styles from "./styles.module.css";
+
 export function Subscribe() {
   return (
-    <div>
-      <div>
-        <h3>Subscribe to Subvisual Inspo</h3>
-        <input placeholder={'Your email'}></input>
-        <button>Subscribe</button>
+    <div className={styles.container}>
+      <div className={styles.imageContainer}>
+        <img className={styles.image} src={'/subscribe-card-image.png'} />
```

Comment:
Something to look into eventually: https://nextjs.org/docs/app/api-reference/components/image
It's a component that handles things like optimization, etc.